analyzer/pkg/dpkg/copyright.go

```diff
@@ -0,0 +1,90 @@
+package dpkg
+
+import (
+	"bufio"
+	"bytes"
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/types"
+	"github.com/aquasecurity/fanal/utils"
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	classifier "github.com/google/licenseclassifier/v2/assets"
+	"io"
+	"regexp"
+	"strings"
+)
+
+const LicenseAdder = "dpkg-license-adder"
+
+var (
+	cl, _                = classifier.DefaultClassifier()
+	copyrightFileRegexp  = regexp.MustCompile(`^usr/share/doc/([0-9A-Za-z_.-]+)/copyright$`)
+	commonLicensesRegexp = regexp.MustCompile(`/?usr/share/common-licenses/([0-9A-Za-z_.+-]+[0-9A-Za-z+])`)
```

Comment:
`commonLicenseReferenceRegexp` as it for lines reference one of common licenses

Reply:
Changed

Reply:
Changed